storm-server/src/main/java/org/apache/storm/localizer/LocalizedResource.java

```diff
@@ -85,26 +83,26 @@
     private final Path baseDir;
     private final Path versionFilePath;
     private final Path symlinkPath;
-    private final boolean uncompressed;
+    private final boolean shouldUnzip;
```

Comment:
nit can we call this `shouldUncompress` or `shouldExpandArchive`?  We support more than just zip.  And this goes everywhere that we have Unzip.

Reply:
If no one disagrees. I was not specific about compression method when I thought about renaming but I see why it might be misleading.